1.9/usage/gpu/index.md

```diff
@@ -0,0 +1,21 @@
+---
+post_title: Enable GPU Acceleration
+nav_title: Enable GPU Acceleration
+feature_maturity: experimental
+menu_order: 110
+---
+
+DC/OS now supports allocating GPUs (Graphics Processing Units) to your long-running DC/OS services. Adding GPUs to your service can dramatically accelerate big data workloads.  [Learn more](http://www.nvidia.com/object/what-is-gpu-computing.html).
+
+# Configure your cluster for GPUs
```

Comment:
Remove this heading since the instructions aren't actually here. Replace with something like `To get started, create a GPU-enabled DC/OS cluster.` And then add hyperlink headings to both sets of instructions. The current topic makes it unclear what the next steps are - the further reading links are most-obvious, but probably least applicable to getting started.